src/commands/schema/import/handler.js

```diff
@@ -1,74 +1,120 @@
 const fs = require('fs');
 const pick = require('lodash/pick');
-const findIndex = require('lodash/findIndex');
-const find = require('lodash/find');
+const pkg = require('../../../../package.json');
+const logger = require('../../../logger');
 
 module.exports = async (argv, ctx) => {
+  // console.log(argv);
   const schema = JSON.parse(fs.readFileSync(argv.file, 'utf8'));
+  if (pkg.version !== schema.version) {
+    logger.warn(`Schema in file '${argv.file}' have different version!`);
```

Comment:
`ctx` already have `logger`, u can use `ctx.logger` instead of very long relative import :)